src/main/resources/hudson/plugins/jira/Messages.properties

```diff
@@ -33,3 +33,4 @@ CommentStep.Descriptor.DisplayName=JIRA: Add a comment to issue(s)
 SearchIssuesStep.Descriptor.DisplayName=JIRA: Search issues
 IssueSelectorStep.Descriptor.DisplayName=JIRA: Issue selector
 JiraCreateIssueNotifier.DisplayName=JIRA: Create issue
+FindIssuesAtCommitLogsStep.Descriptor.DisplayName=JIRA: find all issues referenced by the commit logs of this build
```

Comment:
No, they are shown all consecutive because all step names start with `jira`, not because of the `JIRA: ` fragment in the description. Anyway, I'm nitpicking, you can omit my comment.